data/core/get_started.md

```diff
@@ -160,10 +159,32 @@
     - utter_explain_stack
     - utter_stack_details
     - utter_explain_nlucore
-* affirm OR how_to_get_started{"product":"stack"}
+* affirm OR how_to_get_started{"product":"stack"} OR explain
+    - utter_explain_nlu
+    - utter_explain_core
+    - utter_direct_to_step2
+
+## new to rasa/bots, explain stack, detailed explanation
+
+* how_to_get_started
+    - utter_getstarted
+    - utter_first_bot_with_rasa
+* affirm OR how_to_get_started{"user_type": "new"}
+    - action_set_onboarding
+    - slot{"onboarding": true}
+    - utter_built_bot_before
+* deny
+    - utter_explain_stack
+    - utter_stack_details
+    - utter_explain_nlucore
+* affirm OR how_to_get_started{"product":"stack"} OR explain
     - utter_explain_nlu
     - utter_explain_core
     - utter_direct_to_step2
+* explain
```

Comment:
also not sure it makes sense here, because if the user says "why" to the message "click here if you want to go to the next step" we should be explaining the next step